text-to-speech/v1-generated.ts

```diff
@@ -0,0 +1,1090 @@
+    export enum Voice {
+      EN_US_ALLISONVOICE = 'en-US_AllisonVoice',
+      EN_US_LISAVOICE = 'en-US_LisaVoice',
+      EN_US_MICHAELVOICE = 'en-US_MichaelVoice',
+      EN_GB_KATEVOICE = 'en-GB_KateVoice',
+      ES_ES_ENRIQUEVOICE = 'es-ES_EnriqueVoice',
+      ES_ES_LAURAVOICE = 'es-ES_LauraVoice',
+      ES_LA_SOFIAVOICE = 'es-LA_SofiaVoice',
+      ES_US_SOFIAVOICE = 'es-US_SofiaVoice',
+      DE_DE_DIETERVOICE = 'de-DE_DieterVoice',
+      DE_DE_BIRGITVOICE = 'de-DE_BirgitVoice',
+      FR_FR_RENEEVOICE = 'fr-FR_ReneeVoice',
+      IT_IT_FRANCESCAVOICE = 'it-IT_FrancescaVoice',
+      JA_JP_EMIVOICE = 'ja-JP_EmiVoice',
+      PT_BR_ISABELAVOICE = 'pt-BR_IsabelaVoice'
+    }
+    /** Specify the phoneme set in which to return the pronunciation. Omit the parameter to obtain the pronunciation in the default format. **/
+    export enum Format {
+      IPA = 'ipa',
+      IBM = 'ibm'
+    }
+  }
+
+  /** Parameters for the `synthesize` operation. **/
+  export interface SynthesizeParams {
+    /** Requested audio format (MIME type) of the audio. You can use this header or the `accept` query parameter to specify the audio format. (For the `audio/l16` format, you can optionally specify `endianness=big-endian` or `endianness=little-endian`; the default is little endian.). **/
+    accept2?: SynthesizeConstants.Accept | string;
+    /** Requested audio format (MIME type) of the audio. You can use this query parameter or the `Accept` header to specify the audio format. (For the `audio/l16` format, you can optionally specify `endianness=big-endian` or `endianness=little-endian`; the default is little endian.). **/
+    accept?: SynthesizeConstants.Accept | string;
+    /** Selects a voice to use for synthesis. Retrieve available voices with the `GET /v1/voices` method. **/
+    voice?: SynthesizeConstants.Voice | string;
+    /** GUID of a custom voice model to be used for the synthesis. If a custom voice model is specified, it is guaranteed to work only if it matches the language of the indicated voice. You must make the request with service credentials created for the instance of the service that owns the custom model. Omit the parameter to use the specified voice with no customization. **/
+    customization_id?: string;
+    /** Text to synthesize. **/
+    text: string;
+  }
+
+  /** Constants for the `synthesize` operation. **/
+  export namespace SynthesizeConstants {
+    /** Requested audio format (MIME type) of the audio. You can use this header or the `accept` query parameter to specify the audio format. (For the `audio/l16` format, you can optionally specify `endianness=big-endian` or `endianness=little-endian`; the default is little endian.). **/
+    export enum Accept {
+      BASIC = 'audio/basic',
+      FLAC = 'audio/flac',
+      L16_RATE_NNNN = 'audio/l16;rate=nnnn',
```

Comment:
`____` is the rate value. 
e.g. `audio/l16;rate=16000`

Reply:
The tests are passing, so I am assuming `synthesize()` is functional. As for the user requested audio formats, the user specifies the desired format via either an `accept` query parameter or a `accept2` header parameter.